docs/design/specs/Ecma-335-Augments.md

```diff
@@ -420,3 +420,393 @@ The algorithm is amended as follows:
 **Section** "III.4.2 callvirt" is extended to allow throwing `AmbiguousImplementationException` if the implementation of the interface method resolves at runtime to more than one default interface method. It's also extended to specify throwing `EntryPointNotFoundException` if the default interface implementation is abstract.
 
 **Section** "III.4.18 ldvirtftn" is extended to allow throwing `AmbiguousImplementationException` if the implementation of the interface method resolves at runtime to more than one default interface method. It's also extended to specify throwing `EntryPointNotFoundException` if the default interface implementation is abstract.
+
+## Static Interface Methods
+
+Follow proposed changes to the ECMA standard pertaining to static interface methods. The quotations and page numbers refer to
+version 6 from June 2012 available at:
+
+https://www.ecma-international.org/publications-and-standards/standards/ecma-335/
+
+### I.8.4.4, Virtual Methods
+
+(Add second paragraph)
+
+Static interface methods may be marked as virtual. Valid object types implementing such interfaces shall provide implementations
```

Comment:
Are we going to allow abstract static members?

Reply:
I believe that for now we're only planning abstract static methods i.e. the interface cannot provide a "default implementation" for the methods. As JanK wrote in another comment above:

*I think this was covered in https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-02-24.md#static-abstract-members-in-interfaces: "In order to scope the initial implementation, this proposal intentionally limits such members to only abstract members, not virtual members"*

and DavidWr further clarified that

*Yes. Very specifically, support for default implementation is not allowed. If a static method is virtual, it must be abstract.*